docs/apib/compliancePolicies.apib

```diff
@@ -0,0 +1,214 @@
+FORMAT: 1A
+
+# XClarity compliancePolicies
+
+# get details of all policies
+
+## GET /compliancePolicies
++ Response 200
+{
+   "policies":[{
+      "description":"DEV ONLY",
+      "id":"1427930734934",
+      "inUse":"true",
+      "filterType":"all",
+      "lastModified":"2015-04-01 16:25:34",
+      "name":"DEFAULT-2015-04-01",
+      "updateRule":"",
+      "userDefined":"no"
+   },
+   {"updateRule":"",
+      "description":"ddddd",
+      "id":"1428397839104",
+      "inUse":"false",
+      "filterType":"managed",
+      "lastModified":"This policy was created by USERID on 2015-04-07 05:10:39",
+      "name":"aaaa",
+      "updateRule":"",
+      "userDefined":"yes"
+   }]
+}
++ Response 400
++ Response 401
++ Response 404
++ Response 409
++ Response 500
+
+# retrieve the persisted compare result for servers to which a compliance policy is assigned.
+
+## GET /compliancePolicies/persistedResult
++ Response 200
+{
+   "all": [{
+      "chassis": "FBEF740B178F4EFAA846E7225EE256DC",
+      "cmms": [{
+         "deviceCompliant": ""."message": [],
+         "policyName": "",
+         "supported": "true",
+         "targetFirmware": [],
+         "uuid": "387E51D69EC311E0A4C8E87E4D6C0479"
+      }]
+      "switches": [{
+         "deviceCompliant": "no",
+         "message": [],
+         "policyName": "DEFAULT-2015-04-25",
+         "supported": "true",
+         "targetFirmware": [{
+            "build": "0802",
+            "category": "Switches",
+            "compliant": "no",
+            "componentID": "1B33D6C57ECB04C14567A897DC604900",
+            "componentName": "IO Module 02",
+            "date": "2015-04-22",
+            "fixid": "lnvgy_fw_scsw_si4093-8.2.1.0_anyos_noarch",
+            "level": "higher",
+            "packageExistence": "yes",
+            "reason": "The installed version is down level than the version defined in 
```

Comment:
This line is broken the tests due to JSON format issue

Reply:
Put the entire string in the same line

Reply:
Thanks, I updated it